purchase_order_product_recommendation/README.rst

```diff
@@ -0,0 +1,108 @@
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpurchase--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/purchase-workflow/tree/13.0/purchase_order_product_recommendation
+    :alt: OCA/purchase-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/purchase-workflow-13-0/purchase-workflow-13-0-purchase_order_product_recommendation
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/142/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds a recommended products wizard to current purchase order.
+
+It is based on delivered products to customer locations in a given range of
+dates, and allows the purchase manager to quickly know the most sold products
+for the current supplier, which results in an easy to use hint to improve
+the purchase workflow.
+
+If you want a better mobile usability, the module is ready to use with the
+'web_widget_numeric_step' module. Just install it and you will get a better
+numeric input experience.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Create a new purchase order.
+#. Assign its supplier.
+#. Press *Recommended Products* button.
+#. Set or adjust hinted quantities to order.
+#. Press *Accept*.
+
+The wizard filters products those with supplier infos set for the current order
+supplier. Then it computes how many times those products have been delivered to
+customer locations, and finally it makes a simple estimation of how many
+quantites would be necesary to order given the forcasted stock and the computed
+demand.
+
+If you want to constrain results to only some categories, you can also select
+them in the wizard.
+
+If you have multiple warehouses, you can also constrain the recommendations to
+the deliveries of specific ones.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/purchase-workflow/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/purchase-workflow/issues/new?body=module:%20purchase_order_product_recommendation%0Aversion:%2013.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Tecnativa
+
+Contributors
+~~~~~~~~~~~~
+
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * David Vidal
+  * Ernesto Tejeda
```

Comment:
@manuelcalerosolis You add yourself to contributor's file but I think that after you do not regenerate this file.

Reply:
Done!